src/bandersnatch/default.conf

```
@@ -19,6 +19,11 @@ master = https://pypi.org
 ; catching up for ages.
 timeout = 10
 
+; The global aiohttp coroutine timeout for individual bandersnatch workers.
+; This is set incredibly high by default as individual coroutines need to be
+; equipped to handle mirroring PyPI.
+global-timeout = 10
```

```suggestion
global-timeout = 18000
```
I'm personally more of a 2 hour guy. But what ever - Most good connections should never have to care about this.